PlaybackLab/SerializationDownloadsSampleApp/SerializationDownloadsSampleApp/Views/PlayerViewController.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  PlayerViewController.swift
+//  SerializationDownloadsSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class PlayerViewController: UIViewController {
+  
+  @IBOutlet weak var titleLabel: UILabel!
+  
+  @IBOutlet weak var playerView: UIView!
+  
+  @IBOutlet weak var stateLabel: UILabel!
+  
+  @IBOutlet weak var playOnlineButton: UIButton!
+  
+  @IBOutlet weak var playOfflineButton: UIButton!
+  
+  
+  /**
+   The option contains the info concerning this cell.
+   The cell cares a lot about the embed code to know what to do
+   when it gets the embed code through a notification, for example.
+   */
+  public private(set) var option: PlayerSelectionOption!
```

Comment:
Does it make sense to make `set` private if at the end of the day you make another function to set the option public? It's kinda contradicting.

Reply:
Done, I remove the function and the option is public now.